### PR TITLE
Assignment1

### PR DESCRIPTION
**Pull request by worseTyler**

### CODE REVIEW

**Comment by alexvthornton** [comment]
I will review this

**Comment by alexvthornton** [comment]
Pull Request title includes "Assignment 1" in the title ✔
Pull Request targets Assignment 1 branch ✔
Issue 1: Application no longer crashes ✔
Issue 2: Unit test properly passes ✔
Extra Credit

Issue 3: Feature request implemented ❌
Issue 3: Feature request unit tested ❌

the goal of issue 3 is to randomize the order of the answers, not the questions.

File: PrincessBrideTrivia/PrincessBrideTrivia.Tests/ProgramTests.cs [hunk 1]
```diff
@@ -72,7 +97,37 @@ public void GetFilePath_ReturnsFileThatExists()
...
+        }
+        [TestMethod]
+        public void TestingAssertAreNotEqaul()
+        {
+            string filePath = Program.GetFilePath();
+            string[] one = new string[] { "123" };
+            string[] two = new string[] { "123" };
 
+            Assert.AreNotEqual(one, two);
+        }
```
Comment (worseTyler) [inline PrincessBrideTrivia/PrincessBrideTrivia.Tests/ProgramTests.cs:130]: This was a test to see if MeguminXIII's unit test for his randomization would work, I don't believe it does.

**Comment by worseTyler** [comment]
@alexvthornton Thanks for catching that for me, I read that part of the assignment wrong

**Comment by breaman** [comment]
Nice work on the assignment and on the extra credit.

**Pull request closed without merging**